lamindb/models/artifact.py

```diff
@@ -1798,10 +1812,13 @@ def from_dir(
                 of a registered storage location, the inferred key defaults to `path.name`.
             run: A `Run` object.
 
-        Examples:
-            >>> dir_path = ln.core.datasets.generate_cell_ranger_files("sample_001", ln.settings.storage)
-            >>> artifacts = ln.Artifact.from_dir(dir_path)
-            >>> ln.save(artifacts)
+        Example::
+
+            import lamindb as ln
+
+            dir_path = ln.core.datasets.generate_cell_ranger_files("sample_001", ln.settings.storage)
+            artifacts = ln.Artifact.from_dir(dir_path)
+            ln.save(artifacts)
```

Comment:
We should use `RecordList` here which has `.save()` so that this can go onto a single line, but I fear we're not yet doing this, do we?

Reply:
No, not for `from_dir`